src/azure-cli/azure/cli/command_modules/backup/_params.py

```diff
@@ -87,9 +88,12 @@ def load_arguments(self, _):
     with self.argument_context('backup container show') as c:
         c.argument('name', container_name_type, options_list=['--name', '-n'], help='Name of the container. You can use the backup container list command to get the name of a container.')
         c.argument('backup_management_type', backup_management_type)
+        c.argument('use_secondary_region', arg_type=get_three_state_flag(), help='Use this flag to show container in secondary region. Default: false.')
```

Comment:
This seems to be a command behavior switch, not a resource property. It's recommended to use `action='store_true'` for such cases.

Reply:
Did the changes. Please review once.